Set of changes to start using run_rocm_test.sh for EPSDB.

estewart08 opened the pull request:
Update clone_test.sh to only clone what is needed for EPSDB. Add expected passing lists for babelstream, Nekbone, and openmpapps. Modify various run scripts for validation check.

estewart08 committed:
Set of changes to start using run_rocm_test.sh for EPSDB.

Update clone_test.sh to only clone what is needed for EPSDB.
Add expected passing lists for babelstream, Nekbone, and openmpapps.
Modify various run scripts for validation check.

estewart08 commented:
Currently testing these changes with a ROCm build.

ronlieb reviewed (approved):
assuming it works etc ...

estewart08 committed:
Additional run_rocm_test.sh related updates.

Remove babelstream from default SUITE_LIST until more patches
land in mainline. Add ability to get return code from executable
when using pipe in run_babelstream.sh and delete the additional
run.

estewart08 committed:
[smoke-fails] - Disable printing of runtime fails for EPSDB.
